.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish API documentation
+
+on:
+  workflow_dispatch:
```

Comment:
Plz remove, I assume this was just for testing

Reply:
Yeah good catch, thanks.